WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackBrandingCoordinator.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+/// A class containing convenience methods for the the Jetpack branding experience
+class JetpackBrandingCoordinator {
```

Comment:
Just a suggestion, we could pass in the `viewController` as an unowned instance. So if we then add more functions to the class, we wouldn't need to pass it in each function as I think the VC is a fair shared instance to function. But absolutely up to your taste.

Reply:
Same consideration apply: the PR is still in draft and things like this could change

Reply:
for now I left `JetpackBrandingCoordinator` as a stateless type that uses `static` methods. This can be changed in a future PR if we want to inject the object directly, I just wanted to keep this one more contained in scope.